### PR TITLE
Add 2key2crawl

### PR DESCRIPTION
**Pull request by jrfhoutx**
## Description

Adds support for the 2key2crawl macro pad
changes made in accordance with requests in original commit #4532

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/2key2crawl/keymaps/vol/keymap.c [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+};
+
+void matrix_init_user(void) {
+  debug_config.matrix = 1;
+  debug_config.keyboard = 1;
+  debug_config.enable = 1;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
```
[the remark this answers is not in the document]
Comment (fauxpark) [reply keyboards/2key2crawl/keymaps/vol/keymap.c]: @jrfhoutx you missed this one :)

[the remark this answers is not in the document]
Comment (jrfhoutx) [reply keyboards/2key2crawl/keymaps/vol/keymap.c]: oops. taken care of now, thanks

**Comment by noroadsleft** [comment]
Looks good to me, thanks!

**Comment by jrfhoutx** [comment]
awesome, thanks guys!

**Pull request merged**